doc/conf.py

```diff
@@ -60,6 +60,7 @@
     "gallery_dirs": ["gallery", "tutorials", "projections"],
     "subsection_order": ExplicitOrder(
         [
+            "../examples/gallery/line",
             "../examples/gallery/coast",
             "../examples/gallery/plot",
             "../examples/gallery/grid",
```

Comment:
I'm assuming you want to add another 'line' example (to fully close #493) which is why you've placed in under a separate subsection (instead of under `plot`)?

Reply:
Yes, I plan to add a few more examples for lines in separated PRs. For example:

faults (**plot -Sf**, https://docs.generic-mapping-tools.org/dev/plot.html):
![image](https://user-images.githubusercontent.com/3974108/93031474-25ea5f00-f5f9-11ea-89af-b4b1632e3d8d.png)

quoted lines (**plot -Sq**, https://docs.generic-mapping-tools.org/dev/plot.html)
![image](https://user-images.githubusercontent.com/3974108/93031478-28e54f80-f5f9-11ea-90fe-88ec64ad246c.png)

lines with arrows (https://docs.generic-mapping-tools.org/dev/cookbook/features.html#specifying-line-attributes)
![image](https://user-images.githubusercontent.com/3974108/93031524-8da0aa00-f5f9-11ea-8438-0cbd12f9e8e9.png)

Reply:
Yes, those would be fantastic additions. We can also put in a `velo` plot under this 'line' category once #525 is ready :smile: